readthedocs/api/v2/views/model_views.py

```diff
@@ -381,7 +377,27 @@ class RemoteRepositoryViewSet(viewsets.ReadOnlyModelViewSet):
     pagination_class = RemoteProjectPagination
 
     def get_queryset(self):
-        query = self.model.objects.api(self.request.user)
+        user = (
+            self.request.user
+            if self.request.user.is_authenticated
+            else None
+        )
```

Comment:
If the user is not authenticated, I think we should shortcircuit this and just return an empty queryset, right?